Doc/library/string.rst

```diff
@@ -20,6 +20,12 @@ String constants
 The constants defined in this module are:
 
 
+.. data:: ascii_alphanumerics
+
+   The concatenation of the :const:`ascii_letters` and :const:`digits`
+   constants described below.  This value is not locale-dependent.
+
+     .. versionadded:: 3.11
```

Comment:
Oh please, you have to add empty line also :)

Reply:
Please check indentation is correct

Reply:
I think I've formatted well now. Thanks.